src/java/org/apache/cassandra/sensors/RequestSensorsFactory.java

```diff
@@ -44,4 +62,12 @@ default RequestSensors create(String keyspace)
     {
         return NoOpRequestSensors.instance;
     }
+
+    /**
+     * Create a {@link SensorEncoder} that always return an empty string.
```

Comment:
I would rephrase this javadoc to just specify what this method creates, and separately mention the default returned implementation.